test/agentchat/contrib/test_web_surfer.py

```diff
@@ -44,70 +44,77 @@
 
-    # Try to scroll too far back up
-    response = function_map["page_up"]()
-    assert f"Viewport position: Showing page 1 of {total_pages}." in response
+        # Try to scroll too far back up
+        response = function_map["page_up"]()
+        assert f"Viewport position: Showing page 1 of {total_pages}." in response
 
-    # Try to scroll too far down
-    for i in range(0, total_pages + 1):
-        response = function_map["page_down"]()
-    assert f"Viewport position: Showing page {total_pages} of {total_pages}." in response
+        # Try to scroll too far down
+        for i in range(0, total_pages + 1):
+            response = function_map["page_down"]()
+        assert f"Viewport position: Showing page {total_pages} of {total_pages}." in response
 
-    # Test web search -- we don't have a key in this case, so we expect it to raise an error (but it means the code path is correct)
-    with pytest.raises(ValueError, match="Missing Bing API key."):
-        response = function_map["informational_web_search"](BING_QUERY)
+        # Test web search -- we don't have a key in this case, so we expect it to raise an error (but it means the code path is correct)
+        with pytest.raises(ValueError, match="Missing Bing API key."):
+            response = function_map["informational_web_search"](BING_QUERY)
 
-    with pytest.raises(ValueError, match="Missing Bing API key."):
-        response = function_map["navigational_web_search"](BING_QUERY)
+        with pytest.raises(ValueError, match="Missing Bing API key."):
+            response = function_map["navigational_web_search"](BING_QUERY)
 
-    # Test Q&A and summarization -- we don't have a key so we expect it to fail (but it means the code path is correct)
-    with pytest.raises(AttributeError, match="'NoneType' object has no attribute 'create'"):
-        response = function_map["answer_from_page"]("When was it founded?")
+        # Test Q&A and summarization -- we don't have a key so we expect it to fail (but it means the code path is correct)
+        with pytest.raises(IndexError):
+            response = function_map["answer_from_page"]("When was it founded?")
 
-    with pytest.raises(AttributeError, match="'NoneType' object has no attribute 'create'"):
-        response = function_map["summarize_page"]()
+        with pytest.raises(IndexError):
+            response = function_map["summarize_page"]()
 
 
 @pytest.mark.skipif(
     skip_oai,
     reason="do not run if oai is not installed",
 )
-def test_web_surfer_oai():
-    llm_config = {"config_list": config_list, "timeout": 180, "cache_seed": None}
+def test_web_surfer_oai() -> None:
+    llm_config = {"config_list": config_list, "timeout": 180, "cache_seed": 42}
+
+    # adding Azure name variations to the model list
+    model = ["gpt-3.5-turbo-1106", "gpt-3.5-turbo-16k-0613", "gpt-3.5-turbo-16k"]
+    model += [m.replace(".", "") for m in model]
```

Comment:
`model` looks unused.

Reply:
I'll take a look. 

Generally I think this is a good example of where we should transition to the new model tags feature. It would allow us to not worry about model name variations (use use the OpenAI name as a tag, regardless of what it's called in azure).

Reply:
my mistake, it is fixed now